python/tests/test_highlevel.py

```diff
@@ -486,7 +480,7 @@ def verify_trees(self, ts):
             assert not (st2 != st1)
             left, right = st1.get_interval()
             breakpoints.append(right)
-            self.assertAlmostEqual(left, length)
+            assert left == approx(length)
```

Comment:
I had a good look at this, and I think it'd be better if we used the pattern ``x == pytest.approx(y)``. We're using ``pytest.`` for a bunch of other things, and I don't think there's a good reason for making an exception here. It's more obvious to me what it's actually doing as well, when I see  ``x == pytest.approx(y)`` in isolation (i.e., if I was coming in fresh to this code without reading the imports)

Reply:
Yep, looks like something local. Fixed.